rclcpp/include/rclcpp/executor.hpp

```diff
@@ -413,12 +423,29 @@ class Executor
   is_spinning();
 
 protected:
+  /// Add a node to executor, execute the next available unit of work, and remove the node.
+  /**
+   * Implementation of spin_node_once using std::chrono::nanoseconds
+   * \param[in] node Shared pointer to the node to add.
+   * \param[in] timeout How long to wait for work to become available. Negative values cause
+   * spin_node_once to block indefinitely (the default behavior). A timeout of 0 causes this
+   * function to be non-blocking.
```

Comment:
Probably indent these lines a little further?  At least, that is what is done above.